testsuite/tests/effects/reperform.reference

```diff
@@ -19,4 +19,4 @@
  !3]
  !4]
  !5]
-Unhandled
+Stdlib.Effect.Unhandled_effect(_)
```

Comment:
This `(_)` is unfortunate and reduces the usefulness of including the effect in the exception.  It would be 100% better to also print the name of the effect !  For this we'll need some ad-hoc code in the exception printer(s), I'm afraid.

Reply:
Good point. I see that there is similar code in `printexc.ml`.